### ISSUE TITLE
Chat UI not touch friendly

### ISSUE THREAD

**Issue by adbenitez** [deltachat/deltachat-android-ii#203]
Some buttons are a bit hard to tap, as you can see in the screenshot, the "x" button on the image is small, the edit image (pencil) button is too close to the send button so it is easy to send the image accidentally, the emoji icon's padding is small making it hard to tap.

- Platform (android/blackberry/anbox): Android
- Device: BLU
- Delta Chat Version: 0.97.1
- Screenshots:
![screenshot_20190103-115039](https://user-images.githubusercontent.com/24558636/50660608-6366a000-0f6e-11e9-85e7-411e4a5623af.png)

**Comment by r10s** [comment]
hm, compared to the touch keyboard, the buttons look good to me.

i think also that the real touch-sensitive-area is a bit larger.

**Comment by r10s** [comment]
after targeting #202 and opening the preview when pressing the staged-image, i agree that at least the edit button could be a bit larger.

will also add some space between the staged-image and the text input which will also move the send-button a bit away.

**Comment by adbenitez** [comment]
I think the touching area of the emoji icon could be bigger (look the attached image), there are lot of empty space around it, perhaps it is the screen size of my phone, but the icon is small and hard to tap with my dummy fingers ;) seriously, I usually have to tap it several times before the keyboard shows up, don't know if it is related to the touching area or because the low RAM (458MB) of my phone.

![screenshot_20190104-091252](https://user-images.githubusercontent.com/24558636/50693305-90fe2880-1004-11e9-8ca5-22cd4c0c54c8.png)

**Comment by r10s** [comment]
thanks for explanation :)
maybe we can change the _magin_ of the image to a _padding_, at least for the emoticon/keyboard button.
however, still not sure if this is really needed. did not got reports about too small buttons before :)

**Comment by hpk42** [comment]
On Fri, Jan 04, 2019 at 08:37 -0800, björn petersen wrote:
> thanks for explanation :)
> maybe we can change the _magin_ of the image to a _padding_, at least for the emoticon/keyboard button.
> however, still not sure if this is really needed. did not got reports about too small buttons before :) 

then again, there are not many reporters who give feedback
at this precision/detail level :)

**Comment by adbenitez** [comment]
r10s Well IMO the more dangerous thing was the edit button too close to the send button all the rest are less important details.

**Comment by r10s** [comment]
the edit-button is now larger and a bit more away from the send button:

![image-23](https://user-images.githubusercontent.com/9800740/50702453-45be3700-1051-11e9-8ae0-7ab10bf77b19.png)

**Comment by r10s** [comment]
however, maybe we can move the edit button further into the image. and maybe show it as a round button.

**Comment by adbenitez** [comment]
I don't know anything abut UI design but the edit button could go in the left side so it is close to the hard to tap emiji icon so it is save there ;)

**Comment by adbenitez** [comment]
anyway to avoid user taps the image accidentally while trying to tap the edit/close icons, there are space out of the image, something similar to this can be done, for example:
![edit and close](https://user-images.githubusercontent.com/24558636/50707339-83f62f00-102e-11e9-9ca9-0ec307b6577c.png)

**Comment by r10s** [comment]
this is what i was talking about:

![image-24](https://user-images.githubusercontent.com/9800740/50710309-e79e4d80-106a-11e9-931f-f82df07e81b4.png)

**Comment by adbenitez** [comment]
> tapping the image accidentially is not as worse as tapping the close button accidentially :)

Right, I realized it but after all that GIMP work, decided to upload it anyway XD
I like much more this round edit icon +1

**Comment by r10s** [comment]
sure, thanks for uploading and for all your help. this is very useful. :+1:
